examples/optimizing-programmatically-using-json-example/README.md

```diff
@@ -0,0 +1,95 @@
+## Orchestration Example
+
+An Ionic 5 example to demonstrate how to create Orchestrations with the Thunderhead Cordova Plugin.
+
+### Requirements
+
+The example app was built using the following versions:
+
+```
+Ionic:
+
+   Ionic CLI                     : 6.13.1 (/usr/local/lib/node_modules/@ionic/cli)
+   Ionic Framework               : @ionic/angular 5.5.4
+   @angular-devkit/build-angular : 0.1101.4
+   @angular-devkit/schematics    : 11.2.1
+   @angular/cli                  : 11.1.4
+   @ionic/angular-toolkit        : 3.1.0
+
+Cordova:
+
+   Cordova CLI       : 10.0.0
+   Cordova Platforms : android 9.0.0, ios 6.2.0
+   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.2.1, (and 7 other plugins)
+
+Utility:
+
+   cordova-res : 0.15.3
+   native-run  : 1.3.0
+
+System:
+
+   Android SDK Tools : 26.1.1 (/Users/alexnguyen/Library/Android/sdk/)
+   ios-sim           : 8.0.2
+   NodeJS            : v15.13.0 (/usr/local/Cellar/node/15.13.0/bin/node)
+   npm               : 7.7.6
+   OS                : macOS Big Sur
+   Xcode             : Xcode 12.4 Build version 12D4e
+
+```
+
+### Setup
+
+1. To create Orchestrations in ONE for this tutorial, see this [guide](https://na5.thunderhead.com/one/help/conversations/how-do-i/mobile/ios-orchestrations/one_integrate_mobile_ios_orch_intro/).
```

Comment:
needs to be updated once we have a bespoke one

Reply:
Have we created a ticket for it yet?

Reply:
MOB-1279